tools/sim/start_carla.sh

```diff
@@ -3,7 +3,7 @@
 # Requires nvidia docker - https://github.com/NVIDIA/nvidia-docker
 if ! $(apt list --installed | grep -q nvidia-container-toolkit); then
   read -p "Nvidia docker is required. Do you want to install it now? (y/n)";
-  if [ "${REPLY}" == "y" ]; then
+  if [[ "${REPLY}" == "y" || $CI ]]; then
```

Comment:
is this slow? if not, it's nice to test in CI, otherwise we should put this in the docker image we build in